integration-tests/tests/prisma/mysql/client/default.d.ts

```diff
@@ -0,0 +1 @@
+export * from './index'
```

Comment:
Currently, SingleStore is not compatible with Prisma so, if I'm not mistaken, what this would do is to translate prisma migrations and schemas into Drizzle. I do believe that we can skip this for now